### PR TITLE
Make urls clickable

### PR DESCRIPTION
**Pull request by ghost**
## Describe the change

Added urls to empty links. Fixed the name of a file.

Ref: #18 

## PR checklist

- [ ] I have added description of the change I'm proposing in the OpenMS Documentation.
- [ ] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [ ] I have attached a screenshot of the relevant area after this change.
- [ ] `CHANGELOG.md` is updated.
- [ ] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

**Comment by ghost** [comment]
Screen Shot 2022-05-30 at 10 26 38 am

What urls do we need to add here?

File: docs/faqs/contributor-faq.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ to `TOOLS` if they are stable enough, are fully tested, fully documented, and a
 
 ### I have written a class for OpenMS. What should I do?
 
-Follow the [OpenMS coding conventions]().
+Follow the [OpenMS coding conventions](https://github.com/OpenMS/OpenMS/wiki/Coding-conventions).
```
Comment (tapaswenipathak) [inline docs/faqs/contributor-faq.md:26]: let it be a dead link, for now, this should be in doxygen in API reference.

File: docs/faqs/developer-faq.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ The following section provides general information to new contributors.
 ### I am new to OpenMS. What should I do first?
 
 * Check out the development version of OpenMS (see website).
-* Build OpenMS by following the [installation instructions]() or [from source](../installations/build-openms-from-source.md).
-* Read the [OpenMS Coding Conventions]()
+* Build OpenMS by following the installation instructions on [GNU Linux](../installations/installation-on-gnu-linux.md), [macOS](../installations/installation-on-macos.md), [Windows](../installations/installation-on-windows.md) or [from source](../installations/build-openms-from-source.md).
+* Read the [OpenMS Coding Conventions](https://github.com/OpenMS/OpenMS/wiki/Coding-conventions)
```
Comment (tapaswenipathak) [inline docs/faqs/developer-faq.md:14]: same: https://github.com/OpenMS/OpenMS-docs/pull/61/files#r884384665.

**Comment by tapaswenipathak** [comment]
> What urls do we need to add here?

openms.org.

**Review by tapaswenipathak — COMMENTED** [review]
can you make this ready to be merged to `staging`?

**Comment by ghost** [comment]
This branch has several merge conflicts. I'll make a fresh branch off of staging tomorrow and go through all the URLs.

**Pull request closed without merging**